src/clj/chronograph/middleware.clj

```diff
@@ -13,25 +14,42 @@
         (-> (response/response {:error "Internal Server Error"})
             (response/status 500))))))
 
-
-(defn wrap-authenticated-user
-  [handler]
-  (fn [{:keys [cookies] :as request}]
-    (let [user (some-> cookies
-                       (get "auth-token")
-                       :value
-                       auth/verify-token
-                       :id
-                       users/find-by-id)]
-      (-> request
-          (assoc :user user)
-          handler))))
-
-
-(defn wrap-authorized-user
+(defn wrap-require-user-info
   [handler]
   (fn [{:keys [user] :as request}]
     (if-not user
       (-> (response/response {:error "Unauthorized"})
           (response/status 401))
       (handler request))))
+
+(defn- token-from-bearer-value [bearer-value]
+  (second (string/split bearer-value #" ")))
```

Comment:
seems quite brittle:

```clojure
=> (clojure.string/split "a   b  c   d" #" ")
["a" "" "" "b" "" "c" "" "" "d"]```

will a regex match be better?

Reply:
Don't think it matters, because callers are handling invalid/garbage values. A regex match would provide a better error I think, but that's it.